erpnext/accounts/doctype/chart_of_accounts_importer/chart_of_accounts_importer.py

```diff
@@ -459,8 +459,14 @@ def unset_existing_data(company):
 		as_dict=True,
 	)
 
+	custom_linked = frappe.db.sql(
+		'''select fieldname from `tabCustom Field`
+		where fieldtype="Link" and options="Account" and dt="Company"''',
+		as_dict=True,
+	)
+
 	# remove accounts data from company
-	update_values = {d.fieldname: "" for d in linked}
+	update_values = {d.fieldname: "" for d in linked + custom_linked}
 	frappe.db.set_value("Company", company, update_values, update_values)
```

Comment:
Instead of all of this, just add:

```
from frappe.desk.form.linked_with import get_linked_fields
```

remove the manual SQL and call:

```
fieldnames = get_linked_fields("Account").get("Company", {}).get("fieldname", [])
linked = [{'fieldname': name} for name in fieldnames]
```

Should result in the same, including custom ones, while avoiding direct SQL and repetition.